src/models/networks/unet_bitdiffusion.py

```diff
@@ -1,39 +1,42 @@
 # each layer has a time embedding AND class conditioned embedding
 
+
 class UNet(nn.Module):
     def __init__(
         self,
         dim,
-        init_dim = None,
+        init_dim=None,
         dim_mults=(1, 2, 4, 8),
-        channels = 3,
-        bits = BITS,
```

Comment:
What is `BITS` ? I'm assuming it's meant to be a type hint but it isn't defined anywhere in this script.